test/e2e/form-validation.test.js

```diff
@@ -1,14 +1,126 @@
 const VlFormValidationPage = require('./pages/vl-form-validation.page');
+const { Key } = require('selenium-webdriver');
 
 describe('vl-form-validation', async () => {
-   const vlFormValidationPage = new VlFormValidationPage(driver);
+    const vlFormValidationPage = new VlFormValidationPage(driver);
+    
+    beforeEach((done) => {
+        vlFormValidationPage.load().then(() => {
+            done();
+        });
+    });
+    
+    it('Als gebruiker zie ik dat alle verplichte inputvelden in het eerste en tweede formulier correct gekoppeld zijn met de corresponderende validation-message', async() => {
+        var forms = [1, 2];
+        for (const form of forms) {
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageVoornaam(form), await vlFormValidationPage.getVoornaam(form));
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageNaam(form), await vlFormValidationPage.getNaam(form));
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageEmail(form), await vlFormValidationPage.getEmail(form));
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageIban(form), await vlFormValidationPage.getIban(form));
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageTelefoonnummer(form), await vlFormValidationPage.getTelefoonnummer(form));
+            await verplichtInputElementIsCorrectGekoppeldAanValidationMessage(await vlFormValidationPage.getValidationMessageRijksregisternummer(form), await vlFormValidationPage.getRijksregisternummer(form));
+        }
+    });
+
+    it('Als gebruiker zie ik geen foutmelding als een verplicht veld is ingevuld', async() => {
+        const formZonderSuccessMelding = 1;
+        const validationMessage = await vlFormValidationPage.getValidationMessageVoornaam(formZonderSuccessMelding);
+        const inputField = await vlFormValidationPage.getVoornaam(formZonderSuccessMelding);
+
+        await inputField.setInputValue('Joske');
+        await inputField.sendKeys(Key.TAB);
+
+        await assert.eventually.equal(validationMessage.getErrorMessage(), "");
+        await assert.eventually.isFalse(inputField.hasClass('vl-input-field--error'));
+        await assert.eventually.isFalse(inputField.hasClass('vl-input-field--success'));
+    });
+
+    it('Als gebruiker zie ik een foutmelding als een verplicht veld niet is ingevuld', async() => {
+        const form = 1;
+        const validationMessage = await vlFormValidationPage.getValidationMessageNaam(form);
+        const inputField = await vlFormValidationPage.getNaam(form);
+
+        await inputField.setInputValue('');
+        await inputField.sendKeys(Key.TAB);
```

Comment:
Waarom is dit nodig? Om de validation in werking te laten schieten?
Dat gebeurt enkel wanneer een element out of focus gaat?

Reply:
misschien beter op de knop 'Formulier valideren' klikken ipv te blurren? of beide ...